index.html

```diff
@@ -3,34 +3,32 @@
   <head>
     <meta charset="utf-8">
     <title>Startrly</title>
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="styles/style.css">
   </head>
   <body>
-    <div>
-
-
-      <nav>
-        <ul>
-          <li>
-            Startrly
-          </li>
-          <li>
-            Try Now!
-          </li>
-          <li>
-            About
-          </li>
-        </ul>
-      </nav>
-
-
-      <section>
-        <h1>Startrly</h1>
+    <nav class="navbar">
```

Comment:
All your sectioning attributes could probably be IDs instead of classes, since they're only being used once, but that's just a stylistic thing.